tensorboard/plugins/projector/vz_projector/data.ts

```diff
@@ -528,12 +528,15 @@ namespace vz_projector {
           TSNE_SAMPLE_SIZE
         );
         let labels = new Array(sampledIndices.length);
-        sampledIndices.forEach(
-          (index, i) =>
-            (labels[i] = this.points[index].metadata[
-              superviseColumn
-            ].toString())
-        );
+        sampledIndices.forEach((index, i) => {
```

Comment:
Seems like this `forEach` is actually doing `map`. Can we change it?

Reply:
This whole section was weird, re-wrote